### ISSUE TITLE
FS-UAE may not run smoothly with G-Sync/FreeSync in some cases

### ISSUE THREAD

**Issue by FrodeSolheim** [FrodeSolheim/fs-uae#204]
It is mostly related to a combination of:
- G-Sync not being enabled properly if FS-UAE's vs-sync setting is off
- FS-UAE getting confused about refresh rate if FS-UAE's vs-sync setting is on

So, may need to implement a rendering mode where FS-UAE enables v-sync in OpenGL but also only uses timers for frame rate control (not waits for buffer swap).

(Note, everything works fine with G-Sync / Linux on my computer)

**Comment by bidinou** [comment]
Yep, it happens on my setup. But I'm using the FreeSync compatibility mode of the NVIDIA 430.14 driver (G-Sync over a FreeSync monitor). Thank you !

**Comment by FrodeSolheim** [comment]
The upcoming FS-UAE 4.0 (master branch) should work very nicely with G-Sync and FreeSync displays!
